.pre-commit-config.yaml

```
@@ -1,5 +1,17 @@
 ---
 repos:
+- repo: https://github.com/pre-commit/pre-commit-hooks.git
+  rev: v2.4.0
+  hooks:
+  - id: end-of-file-fixer
+  - id: trailing-whitespace
+    exclude: (test/with-skip-tag-id.yml|examples/example.yml|test/unicode.yml)
```

Let's multiline it for better maintainability:
```suggestion
    exclude: |
      ^x
      (
          test/(with-skip-tag-id|unicode).yml|
          examples/example.yml
      )
```